app/src/main/java/org/mozilla/rocket/home/HomeFragment.kt

```diff
@@ -533,14 +538,17 @@ class HomeFragment : LocaleAwareFragment(), ScreenNavigator.HomeScreen {
     }
 
     private fun showRequestClickContentHubOnboarding(couponName: String) {
-        activity?.let {
-            content_hub.post {
+        val dismissListener = DialogInterface.OnDismissListener {
+            if (isAdded) {
+                restoreStatusBarColor()
```

Comment:
Should we always try to restore status bar color if possible? I found there are null protections in `restoreStatusBarColor()`